.github/workflows/CI.yml

```diff
@@ -28,14 +26,34 @@ jobs:
         id: sdkman
         with:
           candidate: java
-          version: 15.0.0-amzn
+          version: ${{ matrix.java-version }}
       - name: Setup Java
         uses: actions/setup-java@v1
-        id: setup-java
         with:
-          java-version: 15.0.0
+          java-version: 8
           jdkFile: ${{ steps.sdkman.outputs.file }}
       - name: Build and Test
         run: ./gradlew -S --no-daemon --no-parallel build jacocoAggregateReport coveralls
       - name: Lint JavaScript
         run: ./gradlew -S --no-daemon --no-parallel :npm_run_lint-js
+  # TODO merge with SDKMAN version if possible
+  test-across-os:
+    name: '${{ matrix.platform }} with Java 8'
+    strategy:
+      matrix:
+        platform:
+          - ubuntu-latest
+          - windows-latest
+          - macos-latest
+    runs-on: ${{ matrix.platform }}
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+      - name: Setup Java
+        uses: actions/setup-java@v1
+        with:
+          java-version: 8
+      - name: Build and Test
+        run: ./gradlew -S --no-daemon --no-parallel build jacocoAggregateReport coveralls
+      - name: Lint JavaScript
+        run: ./gradlew -S --no-daemon --no-parallel :npm_run_lint-js
```

Comment:
this may be worth revisting.
SDKman should work cross platform on Github Action https://github.com/sdkman/sdkman-action
It's tested across all the same operating system uPortal supports https://github.com/sdkman/sdkman-action/actions/runs/343825892